### ISSUE TITLE
winfrorm sample For Getting Every Frame

### ISSUE THREAD

**Issue by mg021** [Past9/Vlc.DotNet#1]
Hello @Past9 
Can you make 
winform sample For Getting every Frame From Video ????

**Comment by Past9** [comment]
No, but you're welcome to learn VLC and do it yourself on your own repository. This is a fork.
